src/java.base/share/classes/java/util/zip/Deflater.java

```diff
@@ -799,6 +812,8 @@ public int getAdler() {
      * and therefore cannot return the correct value when it is greater
      * than {@link Integer#MAX_VALUE}.
      *
+     * @throws IllegalStateException if the Deflater is closed
```

Comment:
\@throws should follow \@return;  (Style notes in https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html).

Reply:
Done - I've updated the PR to fix this and the other places that were introduced in this PR to follow that style guide.